pkg/transport/generic_transport_controller.go

```diff
@@ -92,6 +106,7 @@ func NewTransportControllerForWrappedObjectGVR(ctx context.Context, bindingInfor
 		wdsClientset:                wdsClientset,
 		wdsDynamicClient:            wdsDynamicClient,
 		wdsName:                     wdsName,
+		bindingCares:                make(map[string]sets.Set[v1alpha1.Destination]),
```

Comment:
wouldn't it be easier to hold a map from destination to set of Binding names? (the opposite direction)
seems to me like a simpler approach since an event that represents a change in customization parameters is for a single destination (and then we have set of binding names right away without any loop/calculation).

Reply:
The direction taken is easier for the code that emits this relation; the opposite direction is easier for the code that consumes this relation.